README.md

```diff
@@ -35,7 +35,8 @@ This will check out the existing directory and run `pulumi preview`.
 The action can be configured with the following arguments:
 
 - `command` (required) - The command to run as part of the action. Accepted
-  values are `up` (update), `refresh`, `destroy` and `preview`.
+  values are `up` (alias: update), `refresh`, `destroy`, `install`, and
```

Comment:
I wonder if rather than an install command, just make command itself optional and if nothing is set then just do the install then exit?
Risk of having a command install is if we ever add `pulumi install` things start clashing (although I'm not sure how likely that is)

Reply:
Yes! That's a great idea! I didn't want to add `install` for that exact reason, but I didn't see a workaround at the time.